oval_graph/client.py

```diff
@@ -118,14 +119,22 @@ def copy_interpreter(self, dst):
             else:
                 shutil.copy2(s, d)
 
+    def get_save_src(self):
+        if isinstance(self.out, str):
```

Comment:
`if self.out` is good enough, although `isinstance` is not wrong. The simpler way to check whether `self.out` is a non-empty string, i.e. "truthy", is more Pythonic.